prisma/schema.prisma

```diff
@@ -93,17 +96,36 @@ model Story {
 model StoryContribution {
   id String @id @default(auto()) @map("_id") @db.ObjectId
 
-  user   User   @relation(fields: [userId], references: [id])
-  userId String @db.ObjectId
+  //user   User   @relation(fields: [userId], references: [id])
+  //userId String @db.ObjectId
+
+  contributor Contributor? @relation(fields: [contributorId], references: [id])
+  contributorId String? @db.ObjectId
 
   story   Story  @relation(fields: [storyId], references: [id], onDelete: NoAction)
   storyId String @db.ObjectId
 
   contributionType ContributionType
+  otherContributorType String?
+  otherContributorCredit String?
+  contributorByline String?
 
   bio String?
 }
 
+model Contributor {
```

Comment:
I think Edward wanted to be able to set a unique "contributor slug" string, as well?

Reply:
Got it, I'll look into this

Reply:
Added